boards/v2/seagate/legend/board.yml

```
@@ -0,0 +1,7 @@
+board:
+  name: legend
+  vendor: seagate
+  revision:
+     format: custom
```

a space too much here:
```suggestion
    format: custom
```

---

Fixed, thanks